test/features/browsersync.js

```diff
@@ -45,7 +45,7 @@ test('it injects the snippet in the right place', t => {
 });
 
 test('it configures Browsersync proxy', t => {
-    t.is(browserSyncConfig().proxy, 'app.test', 'sets default proxy');
+    t.is(browserSyncConfig().proxy, 'laravel-mix.test', 'sets default proxy');
```

Comment:
This test would fail if you don't name the laravel-mix repo folder `laravel-mix` which feels flaky to me. I don't know a good solution to that though.

Reply:
will just do the same and set it to the current folder name. Thanks